cves/CVE-2017-16840.yml

```diff
@@ -127,7 +133,10 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    Since the earliest vcc, the time till the fix was less than a month, about 11 days. 
+    The development of the code was done by one developer and only required a small addition. 
+    About than 10 lines in total across three different files.
```

Comment:
This whole section is not correct. The VCC `ec9e87c922adeffbf4c351d625a4c2e1c430eb19` was committed on February 10, 2016. The fix `3228ac730c11eca49d5680d5550128e397061c85` didn't come until November 8, 2017. That's almost 2 years.